Lib/fontTools/feaLib/ast.py

```diff
@@ -546,8 +546,9 @@ def asFea(self, indent=""):
                 res += " ".join(g.asFea() for g in self.prefix) + " "
             for i, g in enumerate(self.glyphs):
                 res += g.asFea() + "'"
-                if self.lookups[i] is not None:
-                    res += " lookup " + self.lookups[i].name
+                if self.lookups[i]:
+                    for lu in self.lookups[i]:
```

Comment:
The `if` check is superfluous, the loop will do nothing if the list is empty.

Reply:
No, if there's no lookup at that position, then there is an actual None in there that will not iterate.